edgar_hw3/edgar_hw3_7.py

```diff
@@ -0,0 +1,14 @@
+# Дан словарь:
+# {'test': 'test_value',
+# 'europe': 'eur',
+# 'dollar': 'usd',
+# 'ruble': 'rub'}
+# Добавить каждому ключу число равное длине этого ключа (пример {‘key’: ‘value’} ->
+# {‘key3’:‘value’}). Чтобы получить список ключей - использовать метод .keys()
+
+original_dictionary = {'test': 'test_value', 'europe': 'eur', 'dollar': 'usd', 'ruble': 'rub'}
+key_list = list(original_dictionary.keys())
+
+for key in key_list:
```

Comment:
`for key in original_dictionary.keys():`

Reply:
fixed